components/Styles/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export const StyledCateogriesUl = styled.ul`
+  list-style: none;
+  padding: 5px;
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  padding-bottom: 10px;
+`;
+
+export const StyledCategory = styled.li`
```

Comment:
```suggestion
export const StyledCategoryLi = styled.li`
```
would fit your naming style.